app/views/courses/index.html.haml

```diff
@@ -20,6 +21,7 @@
       %td= course.start_date
       %td= course.end_date
       %td= course.allocation
+      %td= course&.institute.name
```

Comment:
if course is nil then this will throw an error. If you never expect course to be nil you can remove the safe operator

```suggestion
      %td= course.institute.name
```

OR

```suggestion
      %td= course&.institute&.name
```

Reply:
Possible N+1 query here, I'm not sure how you're loading `@courses` but you'll need something like

```ruby
@courses = Course.includes(:institutes).all
```

in your controller

Reply:
Thanks - didn't realise about the N+1